### ISSUE TITLE
I want to translate it to chinese language

### ISSUE THREAD

**Issue by yannxia** [dasniko/riding-the-nashorn#2]
may i  translate it ? ☺

**Comment by dasniko** [comment]
I don't think that this is a good idea, b/c Nashorn will be deprecated in the future: http://openjdk.java.net/jeps/335

**Comment by yannxia** [comment]
it's bad news, i should refactor project to groovy,  thank for your article anyhow.
